packages/libcurl.rb

```diff
@@ -41,6 +41,16 @@ class Libcurl < Package
   depends_on 'zlibpkg' # R
   depends_on 'zstd' # R
 
+  def self.patch
+    # Fix arm build error
+    # easy_lock.h:56:7: error: implicit declaration of function 'sched_yield' [-Werror=implicit-function-declaration]
+    # via https://github.com/curl/curl/pull/9054 & https://github.com/curl/curl/pull/9056
+    system 'curl -OLf https://github.com/curl/curl/commit/e2e7f54b7bea521fa8373095d0f43261a720cda0.patch'
+    system 'patch -p1 -i e2e7f54b7bea521fa8373095d0f43261a720cda0.patch'
+    system 'curl -OLf https://github.com/curl/curl/commit/5a1a892565443fa4145888c6150da65c9a33d15c.patch'
+    system 'patch -p1 -i 5a1a892565443fa4145888c6150da65c9a33d15c.patch'
```

Comment:
Use `downloader` with sha256 checksum?

Reply:
done

Reply:
FYI, the filename parameter is not needed unless you want to rename the download.